files/settings-default.cfg

```diff
@@ -259,6 +259,9 @@ texture min filter = linear
 # Texture mipmap type.  (none, nearest, or linear).
 texture mipmap = nearest
 
+# change it to false to fix shadow flickering
+multi thread viewer = true
```

Comment:
You've done something wrong if shadows flicker with multithreading.

Reply:
This flag would allow to debug (it could be cool to diagnose end user issue)

Reply:
Or, as it says in this page I've sent you a link to many times, you could just use the OSG environment variable that does the same thing, as it's unlikely any *user* would ever need to do it, only a developer. https://wiki.openmw.org/index.php?title=Rendering_Architecture#Threading_considerations

Reply:
i'll removed it